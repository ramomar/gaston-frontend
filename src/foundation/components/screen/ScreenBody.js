import React from 'react';
import PropTypes from 'prop-types';
import { Box } from 'grommet';

function ScreenBody(props) {
  return (
    <Box fill='vertical' overflow='scroll'>
      {props.children}
    </Box>
  );
}

ScreenBody.propTypes = {
  children: PropTypes.node.isRequired
};

export default ScreenBody;
