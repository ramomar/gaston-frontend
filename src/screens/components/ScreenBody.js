import React from 'react';
import { Box } from 'grommet';

function ScreenBody({ children }) {
  return (
    <Box>
      {children}
    </Box>
  );
}

export default ScreenBody;
