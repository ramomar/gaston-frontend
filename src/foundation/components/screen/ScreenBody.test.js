import React from 'react';
import renderer from 'react-test-renderer';
import ScreenBody from './ScreenBody';

it('renders correctly', () => {
  const component =
    <ScreenBody>
      {<div>Â¡Hola!</div>}
    </ScreenBody>;

  const tree = renderer.create(component);

  expect(tree).toMatchSnapshot();
});
