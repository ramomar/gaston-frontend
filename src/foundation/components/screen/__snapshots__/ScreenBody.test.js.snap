// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 hCTAHO"
>
  <div>
    Â¡Hola!
  </div>
</div>
`;
